frontend/lib/src/dataframes/arrowFormatUtils.test.ts

```diff
@@ -321,3 +308,67 @@ describe("format", () => {
     ).toEqual('["foo","bar","baz"]')
   })
 })
+
+describe("formatPeriodFromFreq", () => {
+  it.each([
+    // Basic frequencies
```

Comment:
is frequency the official term used in the time libraries' documentations etc.? To me the expected values look more like a timespan than a frequency (which I expect would look something like "every week" or so)

Reply:
I also don't think its the best name, but it is what pandas uses: https://pandas.pydata.org/docs/reference/api/pandas.Period.html#pandas.Period and what we are parsing here is basically just the information from pandas.

Reply:
Alrighty, thank you!